### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by sumit-158**
### What
- Return hello message that help user to navigate to `/docs` to get better understanding to the endpoints

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -4,8 +4,11 @@
 
 app = FastAPI()
 
-@app.get("/brand/president")
+@app.get("/")
+def hello():
+    return {"message": "Hello from facets-knowledge-panels! Tip: open /docs for documentation"}
```
Comment (alexgarel) [inline app/main.py:9]: I don't know if it's useful to speak about /docs here, in production no one will see that folder. People will try to add /docs to the url, leading nowhere….

As currently we only have one url, you could just propose to try it :-)
```suggestion
    return {
      "message": "Hello from facets-knowledge-panels! Tip: open /<facet-name>/<facet-value>. Example: /brand/president"
      "docs": "https://github.com/openfoodfacts/facets-knowledge-panels"
    }
```

Comment (sumit-158) [reply app/main.py:9]: Than let this PR as draft , maybe after some more commit and after having more urls we could merge it.

Comment (alexgarel) [reply app/main.py:9]: sorry I did not realize, FastAPI was generating a documentation by itself, so your first message was perfectly alright !

**Review by alexgarel — APPROVED** [review]
LGTM

**Pull request merged**